### PR TITLE
Reformatted 10 recipes

### PR DESCRIPTION
**Pull request by dash102**
https://codein.withgoogle.com/dashboard/task-instances/4796400818716672/?sp-page=1

### CODE REVIEW

**Comment by waddlesplash** [comment]
@dash102 Please stop opening/closing PRs; if need be, just force-push to overwrite the remote branch. Creating new pull requests just makes our repository messy.

**Comment by dash102** [comment]
Sorry, I didn't know that it makes the repository messy! I couldn't figure out how to correctly squash and amend the commits, so I redid the edits to the recipe files.

**Comment by waddlesplash** [comment]
Read more Git documentation then -- just Google for "git squash commits" or "git soft reset".

**Comment by dash102** [comment]
Ok, thank you, will do research. Do these edits work?

**Comment by waddlesplash** [comment]
No -- the Travis CI build failed -- it's set up to automatically lint things, so that means something in here does not pass lint.

**Comment by dash102** [comment]
Do you know what is wrong with the guilib-1.2.1.recipe to cause the failure? I checked the file and don't see anything wrong with it.

**Comment by waddlesplash** [comment]
Yes, it says:
```
Trailing whitespace in 'haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe'.
```
Pe has a setting to trim trailing whitespace on save; I don't know what editor you're using though.

**Comment by dash102** [comment]
I removed the trailing whitespace, but the check is still failing.

**Comment by waddlesplash** [comment]
Look at the message again:
```
Trailing whitespace in 'haiku-libs/haikuwebkit/haikuwebkit-1.2.1.recipe'.
```
It's right at the bottom of the Travis-CI build output -- can't you see it?

**Comment by dash102** [comment]
@waddlesplash I fixed the trailing whitespace, and the build works now.

**Comment by waddlesplash** [comment]
Alright, can you please squash your commits? We prefer having only one commit instead of 10 for recipes like this.

File: haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe [hunk 1]
```diff
@@ -60,38 +69,21 @@ BUILD_PREREQUIRES="
...
 {
 	cd WebKitBuild/Release
 	make install
-
 	prepareInstalledDevelLibs libWebKit
-
 	# devel package
 	packageEntries devel \
-		$developDir
+	$developDir
```
Comment (waddlesplash) [inline haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe]: This and the BUILD section were correct the way they were before; don't modify them.

File: haiku-libs/haikuwebkit/haikuwebkit-1.2.1.recipe [hunk 2]
```diff
@@ -72,24 +79,9 @@ INSTALL()
 {
 	cd WebKitBuild/Release
 	make install
-
 	rm $developLibDir/*
 	prepareInstalledDevelLibs libWebKit
-
 	# devel package
 	packageEntries devel \
 		$developDir
```
Comment (waddlesplash) [inline haiku-libs/haikuwebkit/haikuwebkit-1.2.1.recipe]: Same here (and below).

File: media-libs/libsdl2/libsdl2-2.0.0.recipe [hunk 3]
```diff
@@ -19,24 +19,33 @@ PROVIDES="
 	lib:libSDL2 = 0.12.0 compat >= 0.12
 	lib:libSDL2_2.0 = 0.12.0 compat >= 0.12
 	"
-
 REQUIRES="
 	haiku$secondaryArchSuffix
-#	lib:libGL
+	lib:libGL
```
Comment (waddlesplash) [inline media-libs/libsdl2/libsdl2-2.0.0.recipe]: This comment was in there on purpose, I believe?

File: media-libs/libsdl2/libsdl2-2.0.0.recipe [hunk 4]
```diff
@@ -19,24 +19,33 @@ PROVIDES="
...
+	libsdl2 == $portVersion base
+	"
+
 BUILD_REQUIRES="
-#	devel:mesa
+	devel:mesa
 	devel:libglu
 	devel:libpng
 	devel:libjpeg
-#	devel:libtiff
```
Comment (waddlesplash) [inline media-libs/libsdl2/libsdl2-2.0.0.recipe]: Same for these two?

**Comment by dash102** [comment]
To squash the commits, do I have to do that in the terminal, or can I do them on github.com? Every time I try to do it in the terminal, my whitespace edits disappear.

**Comment by waddlesplash** [comment]
You must do it in Terminal. What do you mean by whtespace edits disappearing?

**Comment by dash102** [comment]
I did some research and found "git rebase -i HEAD~number" and changed all the "pick"s to "squash"s. Then I pushed the commits, then on github.com, I would get a build failure and have to delete the whitespace again in the files.

**Comment by waddlesplash** [comment]
Try using "git reset" to your first commit (do not add `--hard`!) adding the files (they'll be unstaged changes in the tree) and then committing with --amend.

**Comment by dash102** [comment]
After I do that, do I still do "git rebase..."?

**Comment by waddlesplash** [comment]
No, that is instead of git rebase.

**Comment by dash102** [comment]
Ok, so after that, how do I get those changes on here? Do I do a git push?

**Comment by waddlesplash** [comment]
`git push --force`

**Comment by dash102** [comment]
When I do "git push --force", I go back to having trailing whitespace in the files like it was before I tried to squash the commits.

**Comment by waddlesplash** [comment]
Oh -- that's because you're making the changes on GitHub, and git push --force overwrites them. Fix the trailing spaces locally, --amend again, and then push --force again.

**Comment by dash102** [comment]
I locally reedited the recipe files (I replaced the haikuwebkit ones with new ones), went to the terminal and did: git reset, git commit --amend, git push --force, and I am still getting a failure message.

**Comment by waddlesplash** [comment]
The failure message is still: 
```
Trailing whitespace in 'haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe'.
```
You can enable automatic trimming of trailing whitespace, you know...
Also you still have 8 commits, not 1.

**Comment by dash102** [comment]
How do I enable automatic trimming?

**Comment by waddlesplash** [comment]
You should have been able to find it yourself, but it's in Window -> Preferences -> Files -> Discard trailing space.

**Comment by dash102** [comment]
I have all 10 recipes edited, with no trailing whitespace, but when I do "git push --force", the files are not updated on github.com.

**Comment by dash102** [comment]
I have all 10 recipes edited, with no trailing whitespace, but when I do "git push --force", the files are not updated on github.com.

**Comment by waddlesplash** [comment]
You still have 11 commits. And are you sure you're pushing to the correct branch?

**Comment by dash102** [comment]
I'm trying to use "git push --force". I had 1 commit at one point, but I only had 7 edited recipes at that point, so I'm trying to figure out what I did right that time.

**Comment by dash102** [comment]
@waddlesplash I squashed the commits into 1 commit.

**Comment by waddlesplash** [comment]
I still have 2 comments above, about removing comments in the PROVIDES/REQUIRES sections.

**Comment by dash102** [comment]
@waddlesplash I think that is fixed now.

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**